src/components/common/FilterButton.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { Dispatch, memo } from 'react';
+import styled, { css } from 'styled-components';
+import { useTodosDispatch, useFilterState } from 'context/todoContext/TodoContext';
+import { toggleFilter } from 'context/todoContext/actionCreators';
+import { FilterType, TCategory, TPriority, Action, IFilter } from 'types';
+
+interface FilterButtonProps {
+  type: FilterType;
+  icon: string | JSX.Element;
+  name: TCategory | TPriority;
+}
+
+const FilterButton: React.FC<FilterButtonProps> = ({ type, icon, name }) => {
+  const dispatch: Dispatch<Action> = useTodosDispatch();
+  const filter: IFilter = useFilterState();
+  const isActive: boolean = filter[type].find((f) => f === name) ? true : false;
```

Comment:
`find` 보다는 `includes` 메서드가 더 좋아보입니다! 👍 
```suggestion
  const isActive: boolean = filter[type].includes(name);
```

Reply:
상혁님, 확인해주셔서 감사합니다!! 수정하겠습니다 말씀해주신 코드가 훨씬 좋네요!! 👍👍